static/js/src/imageBuilder.js

```diff
@@ -1,4 +1,37 @@
-(function() {
+(function () {
+  const board_architectures = {
```

Comment:
This is not what we discussed. Duplicating this dictionary into the frontend is horrible.